shared_crates/schema/build.rs

```diff
@@ -13,9 +13,10 @@ fn main() {
             (
                 path.strip_prefix("src")
                     .unwrap()
-                    .to_str()
-                    .unwrap()
-                    .to_owned(),
+                    .iter()
```

Comment:
I do recall us explicitely *not* doing a lossy string conversion to avoid having multiple dirs mapping to the same value if not UTF-8 (in an earlier PR)

Reply:
This is only used to create a lookup table for the schema which we control, so I'm not too concerned.